cylc/flow/task_job_mgr.py

```diff
@@ -268,7 +282,11 @@ def submit_task_jobs(self, suite, itasks, curve_auth,
                     'batch_sys_name': itask.summary['batch_sys_name'],
                 })
                 itask.is_manual_submit = False
-            if is_init == REMOTE_INIT_FAILED:
+
+            if (ri_map[install_target] in [REMOTE_INIT_FAILED,
+                                           REMOTE_FILE_INSTALL_FAILED]):
+                init_error = (ri_map[install_target])
+                del ri_map[install_target]
```

Comment:
I think there was a code comment with the old `del` line which would be helpful here.